components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -758,7 +758,19 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
             deallocate (RossbyRadiusTaper)
          end if
 
-         if (config_Redi_closure == 'constant'.and. &
+         if (config_Redi_closure == 'equalGM') then
+            ! Set Redi closure to be the same as the GM closure, i.e. if
+            ! config_GM_closure is EdenGreatbatch or Visbeck, you will see those
+            ! same closure fields in RediKappa. The horizontal tapering still
+            ! comes from the config_Redi_horizontal_taper flag, not GM.
+            !$omp parallel
+            !$omp do schedule(runtime)
+            do iEdge=1, nEdges
+               RediKappa(iEdge) = RediHorizontalTaper(iEdge) * gmBolusKappa(iEdge)
```

Comment:
if we are doing equalGM shouldn't we use the GMHorizontalTaper instead?  I also find it a little weird to use different tapers.  Are there circumstances we'd expect these to be different?  I guess the flexibility doesn't hurt, but could make configuration more difficult

Reply:
@mark-petersen I'd still suggest we switch the `RediHorizontalTaper` here to `GMhorizontalTaper` this could give some unexpected results if users expect Redi to be exactly equal to GM.

Reply:
@vanroekel, good point. Sorry I missed this before. I made that change and retested.